### ISSUE TITLE
Add option to allow field errors above the field

### ISSUE THREAD

**Issue by paulAcato** [nl-design-system/nlds-gravity-forms#27]
Due to a11y the error messages must be shown above the field. Therefor we want to provide the editor with an option to set it above or below.

**Comment by rianrietveld** [comment]
I asked Gravity Forms the following:

> Placement of the error message
> The NLDS has guidelines for forms (richtlijnen voor formulieren), based on WCAG 2.2 AA, but also on user research.  While writing the NLDS guidelines I always kept in mind: can we do this with Gravity Forms.
> And everything is possible, out of the box, except for one setting: the error message should be above the input field. This is a pattern also used by the [GOV.UK](http://gov.uk/) design system.
> https://design-system.service.gov.uk/components/error-message/
> My question: I saw a new setting for the description placement, is it possible to also add a placement setting for the error message?
